### PR TITLE
Remove and block non-Azure logins for Drupal

### PR DESCRIPTION
**Pull request by weitzman**
- Also ups the autologout duration to be closer to Azure AD duration
- Also disabled basic auth on Tugboat as not recommended anymore https://docs.tugboatqa.com/faq/common-questions/#can-i-password-protect-my-preview-urls
- Uninstalls TFA and password policy modules

**Jira:** (Skip unless you are MA staff)
DP-27097

**To Test:**
- [ ] Make sure login page no longer shows a form

**Followup**
- CX should update the email templates in production. No need to backport those - they are excluded from config sync.
- Possible updates needed to Gitbook

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/sites/default/settings.tugboat.php [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-  $is_endpoint = strpos($_SERVER['REQUEST_URI'], '/api/v1/') !== FALSE;
-  if (!$is_oauth && !$is_endpoint && !(isset($_SERVER['PHP_AUTH_USER']) && ($_SERVER['PHP_AUTH_USER']==$username && $_SERVER['PHP_AUTH_PW']==$password))) {
-    header('WWW-Authenticate: Basic realm="This site is protected"');
-    header('HTTP/1.0 401 Unauthorized');
-    // Fallback message when the user presses cancel / escape
-    echo 'Access denied';
-    exit;
-  }
-}
+//$agent = $_SERVER['HTTP_USER_AGENT'];
```
Comment (arthurbaghdas) [inline docroot/sites/default/settings.tugboat.php:10]: Should we revert this change? @weitzman

Comment (weitzman) [reply docroot/sites/default/settings.tugboat.php:10]: No revert needed. Its deliberate as per https://docs.tugboatqa.com/faq/common-questions/#can-i-password-protect-my-preview-urls.. I decided to comment instead of removing the code just so we have a settings file for future customization.
